File: 백준/Silver/17219. 비밀번호 찾기/비밀번호 찾기.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.util.HashMap;
import java.util.Map;

public class Main {
	public static void main(String args[]) throws Exception {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		
		String input [] = br.readLine().split(" ");
		
		int a = Integer.parseInt(input[0]);
		int b = Integer.parseInt(input[1]);
		
		Map<String, String> map =new HashMap<String, String>();
		
		
		while(a-->0) {
			input = br.readLine().split(" ");
			String site = input[0];
			String password = input[1];
			map.put(site, password);
			
			
		}
		while(b-->0) {
			String s = br.readLine();
			System.out.println(map.get(s));
		}
	}
}

File: 백준/Silver/17219. 비밀번호 찾기/비밀번호 찾기.swift
import Foundation

let nm = readLine()!.split { $0 == " " }.map { Int($0)! }

var dict: [String: String] = [:]
for _ in 0..<nm[0] {
  let input = readLine()!.split { $0 == " " }.map { String($0) }
  dict[input[0]] = input[1]
}

for _ in 0..<nm[1] {
  let input = readLine()!
  if let pw = dict[input] {
    print(pw)
  }
}
